airflow/__init__.py

```diff
@@ -48,3 +48,8 @@
 login: Optional[Callable] = None
 
 integrate_plugins()
+
+
+# Ensure that this query is build in the master process, before we fork of a sub-process to parse the DAGs
+from . import ti_deps
```

Comment:
I don't know if this should be done here or when starting SchedulerJob.  In my opinion, adding additional logic to init is not the best solution and we can probably avoid it in this situation. We don't need this query to be loaded in many cases, e.g. on workers.

Reply:
What do you think about? 
https://github.com/apache/airflow/blob/59fbc2f84a96c1677c8e420d7ce761f1d82b06ac/airflow/jobs/scheduler_job.py#L147

Reply:
(thought I already left this comment, but can't see it)

Yes, this is a hack and needs fixing before this is ready to merge. Putting it somehere in SchedulerJob (or DagFileProcessorAgent perhaps) might be a good way, as that is the only time we need to prime the cache for this specific query.